cmd/clusterctl/client/repository/repository_local_test.go

```diff
@@ -107,28 +110,18 @@ func Test_localRepository_newLocalRepository(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := newLocalRepository(tt.fields.provider, tt.fields.configVariablesClient)
-			if (err != nil) != tt.wantErr {
-				t.Fatalf("error = %v, wantErr %v", err, tt.wantErr)
-			}
-
 			if tt.wantErr {
+				g.Expect(err).To(HaveOccurred())
 				return
 			}
-			if got.basepath != tt.want.basepath {
-				t.Errorf("got.basepath = %v, want = %v ", got.basepath, tt.want.basepath)
-			}
-			if got.providerLabel != tt.want.providerLabel {
-				t.Errorf("got.providerLabel = %v, want = %v ", got.providerLabel, tt.want.providerLabel)
-			}
-			if got.DefaultVersion() != tt.want.defaultVersion {
-				t.Errorf("got.DefaultVersion() = %v, want = %v ", got.DefaultVersion(), tt.want.defaultVersion)
-			}
-			if got.RootPath() != tt.want.rootPath {
-				t.Errorf("got.RootPath() = %v, want = %v ", got.RootPath(), tt.want.rootPath)
-			}
-			if got.ComponentsPath() != tt.want.componentsPath {
-				t.Errorf("got.ComponentsPath() = %v, want = %v ", got.ComponentsPath(), tt.want.componentsPath)
-			}
+
+			g.Expect(err).NotTo(HaveOccurred())
```

Comment:
nit we should be consistent in having assertion on the operation err/on wantErr like in Test_componentsClient_Get